Sources/API/V1/googleapis/google/rpc/error_details.proto

```
@@ -0,0 +1,53 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.rpc;
+
+import "google/protobuf/duration.proto";
```

buf (1.47.2)

Import "google/protobuf/duration.proto" is unused.

(IMPORT_USED)